### PR TITLE
Modify application show action to display searched pets, modify appli…

### PR DESCRIPTION
**Pull request by pkseverance**
- Added "Add Pet to Application" section in the application_show view
- Added search_pets each block to "Add a Pet to Application" section in application show view
- Added params search conditional logic to applications_controller within show method
- Added feature tests for "Add a pet to Application" section
- Added feature tests to search_pets
- Added feature tests for edge cases and sad paths

### CODE REVIEW

File: spec/features/applications/show_spec.rb [hunk 1]
```diff
@@ -88,6 +101,61 @@
...
+        click_button('Submit')
+
+        within('section#add_pet') do
+          expect(page).to have_content("#{@pet_1.name}")
+          expect(page).to have_content("#{@pet_2.name}")
+          expect(page).to have_content("#{@pet_3.name}")
+        end
+      end
+
+      it 'Shows nothing if name != any pets' do
```
Comment (Pocketzs) [inline spec/features/applications/show_spec.rb:147]: I believe this covers most edge cases I can think of including sad path.  Let's continue user stories and play with the rails server local host side to see if we can come up with any more bugs.

**Review by Pocketzs — APPROVED** [review]
